media-api/test/scala/lib/querysyntax/ParserTest.scala

```diff
@@ -498,7 +498,33 @@ class ParserTest extends FunSpec with Matchers with BeforeAndAfter with ImageFie
 
     it("should not match unrelated file types") {
       Parser.run("fileType:catsdogs") should be (List(
-        Match(AnyField, Words("fileType:catsdogs"))
+        Match(SingleField("fileType"), Words("catsdogs"))
+      ))
+    }
+  }
+
+  describe("quoted field search") {
+    it("should match a quoted field search") {
+      Parser.run("\"fieldDogs\":cats") should be (List(
```

Comment:
Triple quotes might be easier to read than escaped quotes.
```suggestion
      Parser.run(""""fieldDogs":cats""") should be (List(
```